descriptions/SDK.iCue.md

```diff
@@ -0,0 +1,3 @@
+
+
+[**CORSAIR iCUE**](https://www.corsair.com/us/en/s/icue-dev) game integrations allow developers to bring games to life like never before. 
```

Comment:
```suggestion
[**CORSAIR iCUE**](https://www.corsair.com/us/en/s/icue-dev) allows games to control rgb lighting of Corsair peripherals.
```
Should be a bit clearer on its purpose (tho feel free to modify it a bit, i'm not a fan of my suggestion)